connectors/django/fma_django/models.py

```diff
@@ -230,6 +237,27 @@ def delete_model_aggregation_schedule_remote(sender, instance, **kwargs):
     _ = events_client.delete_rule(Name=name)
 
 
+def get_aws_event_rule_state(federated_model):
+    """Checks the AWS Event Rule state for a given Federated Model.
+
+    :param federated_model: federated model instance
+    :type federated_model: An instance of a model object with an aggregation task.
+    :return: state of the rule associated with the aggregation task (disabled|enabled)
+    :rtype: (None|string)
+    """
+    state = None
+    try:
+        config = Config(connect_timeout=5, retries={"max_attempts": 0})
```

Comment:
should we allow option to pass args for connect_timeout, retries, region?

Reply:
there is an issue created that abstracts out aws completely as it doesn't really belong in this file.
https://github.com/capitalone/federated-model-aggregation/issues/22

I would imagine that includes having a config file that accounts for this as well

Reply:
We could clarify the issue to reference that these can be settings that would be adjustable in the abstracted scenario.